src/app/courses/course-item/course-item.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-course-item',
+  templateUrl: './course-item.component.html',
+  styleUrls: ['./course-item.component.sass']
+})
+export class CourseItemComponent implements OnInit {
+  @Input() course: Object;
+  @Input() index: Number;
```

Comment:
No need to set `index` here. Usually you'll be using item id to identify specific item.
For example if you implement pagination - then your indexes can be messed up if you try to delete an item.

Reply:
Changed